internal/plugin/provider.go

```diff
@@ -110,6 +110,7 @@ func (p *AivenProvider) Configure(
 func (p *AivenProvider) Resources(context.Context) []func() resource.Resource {
 	return []func() resource.Resource{
 		organization.NewOrganizationResource,
+		organization.NewOrganizationUserGroupMembersResource,
```

Comment:
Hey. This resource must be a beta [beta features](https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices#enabling-beta-features)